pkg/ccl/importccl/import_stmt.go

```diff
@@ -473,6 +488,11 @@ func importPlanHook(
 				return err
 			}
 
+			// TODO(dt): checking *CREATE* on an *existing table* is weird.
```

Comment:
So I thought more about this after our conversation, and I think that checking any privileges is pretty weird here. If I understand correctly, for these statements we require that the user has an admin role, and if the user has admin role than they must have ALL privileges on databases.